src/components/ShowList.jsx

```diff
@@ -1,12 +1,11 @@
 import React from 'react';
 import { useCollection } from 'react-firebase-hooks/firestore';
-import firebase from 'firebase/app';
-import 'firebase/firestore';
-import '../lib/firebase';
+import db from '../lib/firebase';
 
 function ShowList() {
+  const token = localStorage.getItem('token');
   const [value, loading, error] = useCollection(
-    firebase.firestore().collection('items').orderBy('date'),
+    db.collection(token).orderBy('date'),
```

Comment:
I left a comment above about the collection usage. You can refer to that comment and please let me know if you have questions 👍